NBitcoin/RPC/RestClient.cs

```diff
@@ -78,21 +77,24 @@ public RestClient(Uri address, Network network)
 		/// <exception cref="System.ArgumentNullException">blockId cannot be null.</exception>
 		public async Task<Block> GetBlockAsync(uint256 blockId)
 		{
-			if (blockId == null)
-				throw new ArgumentNullException(nameof(blockId));
-
-			var result = await SendRequestAsync("block", RestResponseFormat.Bin, blockId.ToString()).ConfigureAwait(false);

```

Comment:
Keep this one as it was, there might be shitcoins not supporting the verbosity parameter, we don't want to break them.